showcase/src/app/demo/pretty-print/pretty-print.component.html

```diff
@@ -1,7 +1,7 @@
 <example-viewer *ngFor="let fileType of fileTypes; trackBy: trackFileTypes"
 				[extensions]="['HTML', 'SCSS', 'TS']"
 				[filesPath]="fileType.path"
-				[title]="fileType.title">
+				exampleTitle="fileType.title">
```

Comment:
This is not working, it should be [exampleTitle]="fileType.title"

Reply:
Ups, indeed